v2/api/appconfiguration/customizations/configuration_store_extensions.go

```diff
@@ -94,35 +97,31 @@ func secretsSpecified(obj *storage.ConfigurationStore) bool {
 
 	secrets := obj.Spec.OperatorSpec.Secrets
 
-	if secrets.PrimaryConnectionString != nil ||
-		secrets.PrimaryKeyID != nil ||
-		secrets.PrimaryKey != nil ||
-		secrets.SecondaryConnectionString != nil ||
-		secrets.SecondaryKey != nil ||
+	if secrets.PrimaryKeyID != nil ||
 		secrets.SecondaryKeyID != nil ||
-		secrets.PrimaryReadOnlyConnectionString != nil ||
 		secrets.PrimaryReadOnlyKeyID != nil ||
+		secrets.SecondaryReadOnlyKeyID != nil ||
+		secrets.PrimaryKey != nil ||
+		secrets.SecondaryKey != nil ||
 		secrets.PrimaryReadOnlyKey != nil ||
-		secrets.SecondaryReadOnlyConnectionString != nil ||
 		secrets.SecondaryReadOnlyKey != nil ||
-		secrets.SecondaryReadOnlyKeyID != nil {
+		secrets.PrimaryConnectionString != nil ||
+		secrets.SecondaryConnectionString != nil ||
+		secrets.PrimaryReadOnlyConnectionString != nil ||
+		secrets.SecondaryReadOnlyConnectionString != nil {
 		return true
 	}
 
 	return false
 }
 
-func secretsByName(keys []*armappconfiguration.APIKey) map[string]armappconfiguration.APIKey {
-	result := make(map[string]armappconfiguration.APIKey)
-
+func secretsByName(keys []*armappconfiguration.APIKey, result map[string]armappconfiguration.APIKey) {
```

Comment:
Now wondering about the name of this method, given its now a repeated-action, not a transformation. Maybe `addSecretsToMap()`?